test/NuGet.Clients.FuncTests/NuGet.CommandLine.FuncTest/Commands/RestoreCommandTests.cs

```diff
@@ -1326,7 +1326,7 @@ public async Task Restore_WithPackagesConfigProject_PackageWithVulnerabilities_W
                     (new Uri(advisoryUrl2), PackageVulnerabilitySeverity.High, VersionRange.Parse("[1.0.0, 3.0.0)"))
                 });
             pathContext.Settings.RemoveSource("source");
-            pathContext.Settings.AddSource("source", mockServer.ServiceIndexUri);
+            pathContext.Settings.AddSource("source", mockServer.ServiceIndexUri, "true");
```

Comment:
I can't find anything in our coding guidelines document, but generally when a parameter is hard to guess the intent of, to use "named parameter" syntax, so we can see `allowInsecureConnections: "true"`, to make it clear what the value represents.